test/tests/bcs/fixed_value/test.py

```diff
@@ -0,0 +1,30 @@
+"""Tests for imposing BCs in OpenFOAM using MOOSE input file syntax
+"""
+
+import unittest
+import fluidfoam as ff
+import numpy as np
+
+from read_hippo_data import get_foam_times #pylint: disable=E0401
```

Comment:
You could add `test/python` to the `PYTHONPATH` variable in your `.envrc` to get rid of this warning.